assets/javascripts/discourse-assign/routes/group-assignments.js.es6

```diff
@@ -0,0 +1,18 @@
+import { ajax } from "discourse/lib/ajax";
+import Route from "@ember/routing/route";
+
+export default Route.extend({
+  model() {
+    return ajax(`/groups/${this.modelFor("group").get("name")}/members.json`, {
```

Comment:
What happens if a group has lots (e.g. 50+) members. Does this still work?

Reply:
Yes, in the new commit it works with 50+ users too.